src/index.js

```diff
@@ -302,10 +302,34 @@ const addLang = (tree, {rules: {lang}}) => {
   return tree;
 };
 
+const commentsFormatting = tree => {
+  tree.walk = walk;
+  tree.walk(node => {
+    if (typeof node === 'string' && /<!--[^]*-->/gm.test(node.trim())) {
+      const originalComments = node.trim();
+      const content = originalComments.replace(/^<!--/g, '').replace(/-->$/, '');
+      const contentArr = content.split('\n').filter(c => c.trim() !== '');
+
+      if (contentArr.length === 1) {
+        return `<!-- ${contentArr[0].trim()} -->`;
+      }
+
+      const newContent = `<!--\n ${contentArr.join('\n')}\n-->`;
```

Comment:
I think it’s better here to save the contract by simplifying this entry and omit the expression line: 313-315

```js
const contentArr = [
    COMMENT_START, 
    ...content.split('\n').filter(c => c.trim() !== ''), 
    COMMENT_END
];
```
or use `unshift` and `push`

Reply:
cool

Reply:
@Scrum can you check the latest commit whether if that's ok ?